Cargo.toml

```diff
@@ -42,3 +42,6 @@ shadow-rs = { version = "0.36", features = ["metadata"] }
 [features]
 # This feature should be enabled when building static executable
 openssl-vendored = ["openssl/vendored"]
+
+[patch.crates-io]
+gevulot-rs = { git = "https://github.com/gevulotnetwork/gevulot-rs", branch = "feature/reschedule-client-infrastructure" }
```

Comment:
This branch is gonna be removed, right? So probably should merge it first and then update revision here?

Reply:
yes, lets wait for https://github.com/gevulotnetwork/gevulot-rs/pull/41

Reply:
I pinned it to the latest commit on main of gevulot-rs now until we have a release